client/js/analysis.js

```diff
@@ -106,6 +152,16 @@ function removeAllChildNodes(parent) {
   }
 }
 
+/**
+ * Toggles the comment header
+ *
+ */
+function showCommentHeader() {
+  const el = document.getElementById('commentHeader');
+  el.classList.toggle('hidden');
```

Comment:
Provide a value for these toggles so you're not relying on html state being correct when you enter the function:
```
el.classList.toggle('hidden', false);
el.classList.toggle('fade', true);
```